diff --color -Naur glfw-first-patch/src/init.c glfw-patch/src/init.c
--- glfw-first-patch/src/init.c	2022-08-14 22:23:53.386924242 +0200
+++ glfw-patch/src/init.c	2022-08-14 22:24:02.673235186 +0200
@@ -447,7 +447,7 @@
 
     glfwDefaultWindowHints();
 
-    fprintf(stderr, "!!! Patched GLFW from https://github.com/Admicos/minecraft-wayland\n"
+    fprintf(stderr, "!!! Patched GLFW from https://github.com/Admicos/minecraft-wayland (libdecoration)\n"
          "!!! If any issues with the window, or some issues with rendering, occur, "
          "first try with the built-in GLFW, and if that solves the issue, report there first.\n"
          "!!! Use outside Minecraft is untested, and things might break.\n");
